salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -132,6 +132,9 @@ Drain the node:
     - ignore_daemonset: True
     - delete_local_data: True
     - force: True
+    {%- if pillar.orchestrate.get("drain_timeout") %}
+    - timeout: {{ pillar.orchestrate.drain_timeout }}
+    {%- endif %}
```

Comment:
Passing `None` (or `0`, since that's what scripts do) would be simpler and equivalent:

```suggestion
    - timeout: {{ pillar.orchestrate.get("drain_timeout", 0) }}
```

Reply:
True we could default here as well